docs/SQL/development/visual-editor/variant-schema.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Variant schema
+id: variant-schema
+description: About variant schema
+sidebar_position: 2
+tags:
+  - concept
+  - variant
+  - sql
+---
+
+You can use Prophecy to parse your semi-structured data types, and convert them into flat, structured formats to make them easier to understand and use for analytics. This is available for when you want to determine the variant schema of your Snowflake arrays.
+
+Using the variant schema functionality, you can do the following:
+
+- Parse JSON and XML formats
```

Comment:
XML is not supported. We don't have XMLGET supported. We can say JSON here?

Reply:
Done.